features/upgrade/svcat/upgrade.feature

```diff
@@ -0,0 +1,90 @@
+      | p             | {"spec":{"managementState": "Managed"}} |
+      | type          | merge                                   |
+    Then the step should succeed
+    
+  @admin
+  @upgrade-check
+  @users=upuser1,upuser2
+  Scenario: upgrade svcat
+    # Check cluster operator svcat-apiserver status
+    Given the "service-catalog-apiserver" operator version matchs the current cluster version
+    Given the status of condition "Degraded" for "service-catalog-apiserver" operator is: False
+    Given the status of condition "Progressing" for "service-catalog-apiserver" operator is: False
+    Given the status of condition "Available" for "service-catalog-apiserver" operator is: True
+    Given the status of condition "Upgradeable" for "service-catalog-apiserver" operator is: True
+    # Check cluster operator svcat-controller status
+    Given the "service-catalog-controller-manager" operator version matchs the current cluster version
+    Given the status of condition "Degraded" for "service-catalog-controller-manager" operator is: False
+    Given the status of condition "Progressing" for "service-catalog-controller-manager" operator is: False
+    Given the status of condition "Available" for "service-catalog-controller-manager" operator is: True
+    Given the status of condition "Upgradeable" for "service-catalog-controller-manager" operator is: True
+    
+    # Deploy ups broker
+    When I run the :get admin command with:
+      | resource | clusterservicebroker |
+    Then the step should succeed
+    Given admin ensures "ups-broker" clusterservicebroker is deleted after scenario  
+    Given I have a project
+    When I switch to cluster admin pseudo user
+    And I use the "<%= project.name %>" project
+    When I process and create:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/svc-catalog/ups-broker-template.yaml |
+      | p | UPS_BROKER_PROJECT=<%= project.name %>                                                                  |
+    Then the step should succeed
+    And I wait up to 300 seconds for the steps to pass:
+    """
+    When I run the :describe client command with:
+      | resource | clusterservicebroker/ups-broker |
+    Then the output should contain "Successfully fetched catalog entries from broker"
+    """
+    When I run the :get client command with:
+      | resource | clusterserviceclass                                                       |
+      | o        | custom-columns=CLASSNAME:.metadata.name,EXTERNAL\ NAME:.spec.externalName |
+    Then the output should contain "user-provided"
+
+    #Provision a serviceinstance
+    Given I switch to the first user
+    When I process and create:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/svc-catalog/ups-instance-template.yaml |
+      | p | USER_PROJECT=<%= cb.user_project %>                                                                       |
```

Comment:
Didn't find where is `user_project` saved. We can use `<%= project.name %>` here directly.